packages/idd/idd.0.1/opam

```diff
@@ -0,0 +1,20 @@
+opam-version: "2.0"
+name: "idd"
+depends: [
+  "ocaml" {>= "4.04.0"}
+  "dune" {build}
+  "base" {>= "0.12.0"}
+  "ppx_jane"
+  "ppx_deriving"
+]
+build: ["dune" "build" "-p" name]
```

Comment:
```suggestion
build: ["dune" "build" "-p" name "-j" jobs]
```
you might want to allow users to control the number of jobs they use for the compilation of this package